### DISCUSSION TITLE
Getting rid of chrome window titlebar

### DISCUSSION THREAD

**Discussion by wolfsburg13** [najdek/f1tv_plus#8] [Ideas]
Is there a way to get rid of the URL bar+title bar from the video when in layout mode? Soooo much space taken by URLs and Titlebars

Thanks! @najdek

**Comment by najdek** [comment]
It's possible to hide URL bars in Chrome.
You'll need to create a shortcut to F1TV home page:
Chrome menu (3 dots) -> "More Tools" -> "Create shortcut" -> Check "Open as window".

I plan to add an option to play multiple streams in one window, so there will be no screen space wasted. It will look like this:
![2021-09-07 215635](https://user-images.githubusercontent.com/4890607/132403625-bc0a6691-5d3c-4402-a307-d8889b63ef72.png)

**Comment by najdek** [comment]
You can now play multiple streams in one browser window. 
Get latest update here: https://github.com/najdek/f1tv_plus/raw/master/f1tv_plus.user.js

**Reply by MiSeRy81** [reply]
THIS update is awesome. Thank you very much.
